libraries/python/mcp-tunnel/mcp_tunnel/_main.py

```diff
@@ -229,30 +230,34 @@ def write_assistant_config(servers: list[MCPServer], tunnel: MCPTunnel) -> None:
             task_completion_estimate: 30
     """
 
-    config = {
-        "tools": {
+    personal_mcp_servers = []
+    for server, port in zip(servers, tunnel.ports):
+        server_config = {
+            "key": server.name,
             "enabled": True,
-            "personal_mcp_servers": [
-                {
-                    "key": server.name,
-                    "enabled": True,
-                    "command": port.sse_url,
-                    "args": [
-                        json.dumps({
-                            "tunnel_id": port.tunnel_id,
-                            "port": port.port,
-                            "access_token": tunnel.access_token,
-                        })
-                    ],
-                    "prompt": "",
-                    "long_running": False,
-                    "task_completion_estimate": 30,
-                    **server.extra_assistant_config,
-                }
-                for server, port in zip(servers, tunnel.ports)
+            "command": port.sse_url,
+            "args": [
+                json.dumps({
+                    "tunnel_id": port.tunnel_id,
+                    "port": port.port,
+                    "access_token": tunnel.access_token,
+                })
             ],
+            "prompt": "",
+            "long_running": False,
+            "task_completion_estimate": 30,
+            **server.extra_assistant_config,
         }
-    }
+
+        # Special handling for the filesystem-edit server
+        if server.name == "mcp-server-filesystem-edit" and sys.platform.startswith("win32"):
```

Comment:
Don't put this logic in MCP-tunnel. Put it in bundle. It is passed via extra_assistant_config.